geoengine_geoname_geocoder/geonames_coder.py

```diff
@@ -33,11 +33,16 @@
 
 logger = logging.getLogger('GeoNames address encoding')
 
+
 class ResPartner(osv.osv):
```

Comment:
Also, please remove `ResPartner()` at the end of the file, both are deprecated forms